timelock/cli/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
```

Comment:
Maybe just remove this cli entirely from the review?  The program itself is enormous and probably should be reviewed in isolation

Reply:
Done